README.md

```diff
@@ -75,6 +75,12 @@ The poller emits an `error` event on errors from the Split.io API.
 that adds serialized data to the `window.__splitCachePreload` object. The
 serialized data will be used to determine cohort allocations.
 
+`generateSerializedDataScript` accepts the following arguments:
+
+| Property                      | Description |
+|-------------------------------|-------------|
+| splits | Array of strings that, if included, filters the splitsData (Optional) |
```

Comment:
i think we should call this `experiments` to match the language we use in the SDKs

Reply:
hm, actually i don't know. it would be odd in the SDKs to ask partners to pass in `experiments` and `splits` when they are the same thing -- that's what i think we should avoid. but maybe `splits` makes more sense here since it's split.io-specific

Reply:
yea I was planning for `experiments` in the SDK, but `splits` here to keep with the split theme